doc/distrib/NodeHelpFiles/CoreNodeModels.DefineData.md

```diff
@@ -1,23 +1,26 @@
 ## In Depth
 
-The Define Data node validates the data type of incoming data. It can be used to ensure local data is of the desired type and is also designed to be used as an input or output node, declaring the type of data a graph expects or provides. The node supports a selection of commonly used Dynamo data types, for example 'String', 'Point', or 'Boolean'. The full list of supported data types is available in the drop-down menu of the node.
```

Comment:
seems like your PR might be making changes it should not be making?

Reply:
Or is this just showing commits there are already in master?

Reply:
@mjkkirschner yep, those commits are in master already